right-for-life_front/src/app/App.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      animals: [],
+      textId: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:4000/animals')
```

Comment:
You may use BE_URL constant for http://localhost:4000 part

Reply:
missed! thnx!